packages/atomic/src/utils/utils.ts

```diff
@@ -96,3 +97,15 @@ export function elementHasAncestorTag(
   if (parentElement.tagName === tagName.toUpperCase()) return true;
   return elementHasAncestorTag(parentElement, tagName);
 }
+
+export function sanitizeStyle(style: string) {
+  const purifiedOuterHTML = sanitize(`<style>${style}</style>`, {
+    ALLOWED_TAGS: ['style'],
+    ALLOWED_ATTR: [],
+    FORCE_BODY: true,
```

Comment:
`FORCE_BODY` is necessary to sanitize this. Otherwise, an empty string is returned.

Excerpt from their doc:
```js
// glue elements like style, script or others to document.body and prevent unintuitive browser behavior in several edge-cases (default is false)
var clean = DOMPurify.sanitize(dirty, {FORCE_BODY: true});
```